src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -60,6 +60,10 @@
  * the operating system.</li>
  * </ol>
  *
+ * <p>In contrast to {@link #exit exit}, the {@link #halt halt} method does not initiate the
+ * shutdown sequence. See the <a href="#termination">Java Virtual Machine Termination</a> section
+ * below.
```

Comment:
The forward ref seems inappropriate when this is the section describing which events trigger the shutdown sequence.